manifests/server/options.pp

```diff
@@ -20,6 +20,7 @@
 define dns::server::options(
   $forwarders = [],
   $listen_on = [],
+  $listen_on_port = undef,
```

Comment:
Needs docstring at least.
We haven't been good at this, but can you add it per the format specified here?
https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc